### PR TITLE
bugzilla: verify that all linked PRs as reviewed by QE

### PR DESCRIPTION
**Pull request by AlexNPavel**
This PR changes the behavior of the `release-controller` bugzilla
verifier to ensure that all PRs linked to a bug have been reviewed with
a `/lgtm` by the QE contact.

/cc @bradmwilliams @stevekuznetsov

### CODE REVIEW

File: pkg/bugzilla/bugzilla.go [hunk 1]
```diff
@@ -36,9 +36,8 @@ func NewVerifier(bzClient bugzilla.Client, ghClient github.Client, pluginConfig
 	}
 }
 
-// pr contains a bugzilla bug ID and the associated GitHub pr that resolves the bug
-type pr struct {
-	bugID int
+// PR contains the org, repo, and PR number for a PR
+type PR struct {
```
Comment (stevekuznetsov) [inline pkg/bugzilla/bugzilla.go:40]: Exposing a struct but none of it's fields is weird - does this thing have exported methods?

File: pkg/bugzilla/bugzilla.go [hunk 2]
```diff
@@ -56,27 +55,33 @@ var (
...
-		comments, err := c.ghClient.ListIssueComments(bzp.org, bzp.repo, bzp.prNum)
-		if err != nil {
-			errs = append(errs, fmt.Errorf("Unable to get comments for github pull %s/%s#%d: %v", bzp.org, bzp.repo, bzp.prNum, err))
-			continue
-		}
-		reviews, err := c.ghClient.ListReviews(bzp.org, bzp.repo, bzp.prNum)
-		if err != nil {
-			errs = append(errs, fmt.Errorf("Unable to get reviews for github pull %s/%s#%d: %v", bzp.org, bzp.repo, bzp.prNum, err))
-			continue
+		// TODO: use this list to comment on bugzilla bugs with reason for not auto-verifying a PR
```
Comment (stevekuznetsov) [inline pkg/bugzilla/bugzilla.go:67]: Why leave this TODO?

Comment (AlexNPavel) [reply pkg/bugzilla/bugzilla.go:67]: I was planning to do all the bugzilla commenting work as part of a separate PR after this one is merged. This `TODO` was put there mostly to indicate why I changed this to an array when I could have left it as a boolean as before.

File: pkg/bugzilla/bugzilla.go (hunk 1 above)
Comment (AlexNPavel) [reply pkg/bugzilla/bugzilla.go]: No, it doesn't need to be exported. Changed it back.

**Review by stevekuznetsov — APPROVED** [review]

**Pull request merged**